### PR TITLE
Feature privacy with blockvault test

### PR DESCRIPTION
**Pull request by shahabi8**
This PR creates a Python test for feature privacy with blockvault. There is a nodeos cluster with three nodes, two of them has access to blockvault. Security group is introduced to restrict non-participant node access to produced blocks as well as change of password for postgres database such that the non-participant nodes cannot retrieve data from blockvault.

## Change Type
**Select *ONE*:**
- [x] Documentation

- [ ] Stability bug fix

- [ ] Other

- [ ] Other - special case

## Testing Changes
**Select *ANY* that apply:**
- [x] New Tests

- [ ] Existing Tests

- [ ] Test Framework

- [ ] CI System

- [ ] Other

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [x] Documentation Additions
It should be mentioned that security group does not restrict non-participant nodes access to blockvault. Postgres login credentials need to get updated to restrict non-participant nodes access.

### CODE REVIEW

File: tests/feature_privacy_blockvault.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+testSuccessful=False
+killEosInstances=not dontKill
+killWallet=not dontKill
+
+WalletdName=Utils.EosWalletName
+ClientName="cleos"
+
+try:
+    TestHelper.printSystemInfo("BEGIN")
+    cluster.setWalletMgr(walletMgr)
```
Comment (dimas1185) [inline tests/feature_privacy_blockvault.py:60]: you can save 2 lines and initialize wallet manager inside Cluster constructor

File: tests/feature_privacy_blockvault.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+    Print("#################################################################################")
+    Print("# Scenario 2: Adding node 0 and node 1 to a security group, check access of node 2 and node 0 to blocks produced by node 1#")
+    Print("#################################################################################")
+
+    Print("Add nodes 0 and 1 to security group")
+    securityGroup.editSecurityGroup(addNodes=[node0, node1])
+    securityGroup.verifySecurityGroup()
+
+    Print("### change postgre password to restrict node 2 access to it ###")
```
Comment (dimas1185) [inline tests/feature_privacy_blockvault.py:123]: my understanding of blockvault feature is that it is used to create a cluster with same producer key with backup instances. Hence all of it's instances supposed to have same password and same certificate. So I don't see much logic in the scenario you are testing. Correct me if I missed something.

**Review by dimas1185 — CHANGES_REQUESTED** [review]
two things:
1. please add more comments, test description, similar to the one in PR and ideally comment every step. this is essential for other people to maintain your test or solve flakyness if it will appear.
2. please review your test scenario, see my comment on line 123

**Comment by shahabi8** [comment]
Closed this PR. Blockvault is not supposed to use this way.

**Pull request closed without merging**